### PR TITLE
Android - Add admob ads

### PR DESCRIPTION
**Pull request by RodriSanchez1**
Needs to add admob plugin and cordova-android-play-services-gradle-release plugin [Ccboard PR](https://github.com/cboard-org/ccboard/pull/28)

### CODE REVIEW

File: src/components/Board/Board.container.js [hunk 1]
```diff
@@ -276,6 +276,9 @@ export class BoardContainer extends Component {
     this.setState({ isFixedBoard: !!boardExists.isFixed });
 
     // if (isAndroid()) downloadImages();
+    if (isAndroid()) {
+      await this.handleAdMobAds();
```
Comment (tomivm) [inline src/components/Board/Board.container.js]: I think that is not necessary to await during ad loading. Considering that is not showing on this component.

File: src/components/Board/Board.container.js [hunk 2]
```diff
@@ -360,6 +363,14 @@ export class BoardContainer extends Component {
     }
   }
 
+  async handleAdMobAds() {
+    const { bannerAd, interstitialAd } = adMobAds;
+    if (bannerAd._created) await bannerAd.hide();
+    await interstitialAd
+      .load()
+      .catch(msg => console.log('Error load interstitial Ad', msg));
```
Comment (martinbedouret) [inline src/components/Board/Board.container.js]: use `console.error` instead

File: src/components/Settings/Settings.container.js [hunk 3]
```diff
@@ -16,6 +17,17 @@ export class SettingsContainer extends Component {
     logout: PropTypes.func.isRequired,
     isDownloadingLang: PropTypes.bool
   };
+  async componentDidMount() {
+    if (isAndroid()) {
+      const { bannerAd, interstitialAd } = adMobAds;
+      interstitialAd
+        .show()
+        .catch(msg => console.log('The interstitial advice is not available'));
```
Comment (martinbedouret) [inline src/components/Settings/Settings.container.js]: use `console.error` instead

File: src/components/Settings/Settings.container.js [hunk 4]
```diff
@@ -16,6 +17,17 @@ export class SettingsContainer extends Component {
...
+      bannerAd
+        .show()
+        .catch(msg => console.log('The banner advice is not available'));
```
Comment (martinbedouret) [inline src/components/Settings/Settings.container.js]: use `console.error` instead

File: src/cordova-util.js [hunk 5]
```diff
@@ -40,6 +40,13 @@ export const initCordovaPlugins = () => {
     } catch (err) {
       console.log(err.message);
     }
+    if (isAndroid()) {
+      try {
+        prepareAds();
+      } catch (err) {
+        console.log(err.message);
```
Comment (martinbedouret) [inline src/cordova-util.js]: use `console.error` instead

**Review by martinbedouret — CHANGES_REQUESTED** [review]

**Review by martinbedouret — APPROVED** [review]

**Comment by RodriSanchez1** [comment]
resolve #1204

**Pull request merged**